pkg/apis/eksctl.io/v1alpha5/access_entry.go

```diff
@@ -6,16 +6,20 @@ import (
 	"strings"
 
 	"github.com/aws/aws-sdk-go-v2/aws/arn"
+	ekstypes "github.com/aws/aws-sdk-go-v2/service/eks/types"
 )
 
 // AccessEntry represents an access entry for managing access to a cluster.
 type AccessEntry struct {
+	// existing IAM principal ARN to associate with an access entry
 	PrincipalARN ARN `json:"principalARN"`
-	// +optional
```

Comment:
Not very sure here but i think `// +optional` are supposed to be a one line comment. They are parsed by controller-gen if i'm not mistaken..

Reply:
See https://github.com/kubernetes-sigs/controller-tools/pull/211/files#diff-3b88366c37358db48ac32a74f079eb2d1c89727c3cd99ddb552908a05794c94fR66-R70

Reply:
Do we use `controller-gen` to generate CRDs for eksctl?